sdks/python/setup.py

```diff
@@ -200,6 +200,8 @@ def get_version():
     'google-cloud-language>=1.3.0,<2',
     'google-cloud-videointelligence>=1.8.0,<2',
     'google-cloud-vision>=0.38.0,<2',
+    # GCP Package required by Google Cloud Profiler.
+    'google-cloud-profiler>=3.0.4,<4'
```

Comment:
do you want to install some version in this range  in https://github.com/apache/beam/blob/e52e9e4d05cecbe00264e81e089131a69cc2112c/sdks/python/container/base_image_requirements.txt#L43 ?

Reply:
good point.